src/app/modules/report-module/components/chart-wrapper/index.tsx

```diff
@@ -189,7 +189,7 @@ export function ReportChartWrapper(props: Props) {
         </h4>
       )}
       <CommonChart
-        withHeader
+        // withHeader={false}
```

Comment:
either remove the commented code, or explain why it is commented to reduce tech debt